easybuild/easyconfigs/a/Arrow/Arrow-11.0.0-gfbf-2022b.eb

```diff
@@ -11,7 +11,11 @@ toolchain = {'name': 'gfbf', 'version': '2022b'}
 
 source_urls = ['https://archive.apache.org/dist/%(namelower)s/%(namelower)s-%(version)s']
 sources = ['apache-arrow-%(version)s.tar.gz']
-checksums = ['2dd8f0ea0848a58785628ee3a57675548d509e17213a2f5d72b0d900b43f5430']
+patches = ['Arrow-8.0.0_bugfix.patch']
```

Comment:
@PetrKralCZ The patch filename should be a bit more descriptive, "bugfix" is way too generic.

How about `Arrow-8.0.0_fix-BaseExtensionType-arrow-ext-methods.patch`